HabboHotel/Users/HabboStatsService.cs

```diff
@@ -0,0 +1,69 @@
+using Dapper;
+using Plus.Database;
+
+namespace Plus.HabboHotel.Users;
+
+public class HabboStatsService : IHabboStatsService
+{
+    private readonly IDatabase _database;
+
+    public HabboStatsService(IDatabase database) => _database = database ?? throw new ArgumentNullException(nameof(database));
+
+    public async Task<HabboStats> LoadHabboStats(int userId)
+    {
+        using var connection = _database.Connection();
+
+        var statRow = await connection.QueryFirstOrDefaultAsync<dynamic>(

```

Comment:
Might want to query a `HabboStats` object instead of a `dynamic`. Then there is no need to do parsing / conversion.